packages/plugin-flex/src/sub-commands/flex-plugin.ts

```diff
@@ -463,15 +481,32 @@ export default class FlexPlugin extends baseCommands.TwilioClientCommand {
       flexConfigOptions,
     );
     this._serverlessClient = new ServerlessClient(this.twilioClient.serverless.v1.services, this._logger);
+    this._telemetry = new Telemetry();
 
     if (!this.isJson) {
       this._logger.notice(`Using profile **${this.currentProfile.id}** (${this.currentProfile.accountSid})`);
       this._logger.newline();
     }
+    const start = performance.now();
     const result = await this.doRun();
+    const end = performance.now();
+
     if (result && this.isJson && typeof result === 'object') {
       this._logger.info(JSON.stringify(result));
     }
+
+    if (this.getTopicName() === validaTopicName || this.getTopicName() === deployTopicName) {
```

Comment:
Suggestion: Instead of this, we can set the payload for track in individual command classes and remove this business logic and checks. If the payload is set, we will append that to the common payload and pass to this.track.